crates/nu-parser/tests/test_parser.rs

```diff
@@ -2422,6 +2499,19 @@ mod mock {
         ) -> Result<PipelineData, ShellError> {
             todo!()
         }
+
+        fn is_const(&self) -> bool {
+            true
+        }
+
+        fn run_const(
+            &self,
+            _working_set: &StateWorkingSet,
+            _call: &Call,
+            _input: PipelineData,
+        ) -> Result<PipelineData, ShellError> {
+            panic!("Should not be called!")
```

Comment:
cont.: At the same time this could be a good canary that const eval is truly not executed.